### PR TITLE
enh(UX) : host discovery mappers duplication

### PR DESCRIPTION
**Pull request by Med1233**
## Description

fix popups  overflow 

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/packages/ui/src/ActionsList/index.tsx [hunk 1]
```diff
@@ -28,12 +29,16 @@ const useStyles = makeStyles()({
...
+  className,
+  actions,
+  onClick: onMenuListClick
+}: Props): JSX.Element => {
   const { cx, classes } = useStyles();
   const { t } = useTranslation();
 
   return (
-    <MenuList className={cx(classes.list, className)}>
+    <MenuList className={cx(classes.list, className)} onClick={onMenuListClick}>
```
Comment (bdauria) [inline centreon/packages/ui/src/ActionsList/index.tsx:41]: I would change the naming the other way around: let onClick be in this scope, and name onClick: onActionClick below

File: centreon/packages/ui/src/ActionsList/index.tsx [hunk 2]
```diff
@@ -28,15 +29,15 @@ const useStyles = makeStyles()({
...
 
-const ActionsList = ({ className, actions }: Props): JSX.Element => {
+const ActionsList = ({ className, actions, onClick }: Props): JSX.Element => {
   const { cx, classes } = useStyles();
   const { t } = useTranslation();
 
   return (
-    <MenuList className={cx(classes.list, className)}>
-      {actions?.map(({ Icon, label, onClick }) => {
+    <MenuList className={cx(classes.list, className)} onClick={onClick}>
```
Comment (Thebarda) [inline centreon/packages/ui/src/ActionsList/index.tsx:37]: Why do you want a click listener on MenuList?

Comment (Med1233) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: there is an overflow between the two popups. I want to close the action popup while the confirm/duplicate popup is shown

Comment (Thebarda) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: Ok but it's related to the Menu component

Comment (Med1233) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: the prop onClick can be used for any action

Comment (Thebarda) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: In you can, you want to close the popup while the modal is displaying, right?

Comment (Med1233) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: yes

Comment (Thebarda) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: Ok so you don't need the onClick. You just have to set a state that displays the modal and fades away the Menu popup

Comment (Med1233) [reply centreon/packages/ui/src/ActionsList/index.tsx:37]: yes, you are right

**Pull request closed without merging**